modules/material-management/src/keyring.ts

```diff
@@ -50,7 +50,7 @@ export abstract class Keyring<S extends SupportedAlgorithmSuites> {
      */
     needs(material === _material, 'New EncryptionMaterial instances can not be created.')
 
-    /* Postcondition: If this keyring generated data key, it must be the right length.
+    /* Postcondition UNTESTED: If this keyring generated data key, it must be the right length.
```

Comment:
Why is this untested? Is there a reason this is not testable, or is adding the test outside the scope of this PR?

Reply:
This is covered by the materials themselves.  The comment is there to make sure that people know it needs to be verified.  But there is no code to be tested...